provisioner/docker/config_openeuler-22.03.yaml

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker:
+        memory_limit: "4g"
+        image: "bigtop/slaves:trunk-openeuler-22.03"
+        #repo: "http://repios.bigtop.apache.org/releases/3.1.0/openEuler/22.03/$basearch"
+repo: "file:///bigtop-home/output/"
```

Comment:
Please remove this local repo. 
We don't need add local repo:`file:///bigtop-home/output` If `enable_local_repo`is set to `true`.

Reply:
ok, the modify commit:  31cbe9ed85762dcc8a13d3872d00b09895f24ab9

Reply:
There may be some ambiguities in my words.
What I mean is to **uncomment** `#repo: "http://repios.bigtop.apache.org/releases/3.1.0/openEuler/22.03/$basearch"`, not remove it.

e.g. from Fedora36:
```
docker:
        memory_limit: "4g"
        image: "bigtop/puppet:trunk-fedora-36"

repo: "http://repos.bigtop.apache.org/releases/3.2.0/fedora/36/$basearch"
distro: centos
components: [hdfs, yarn, mapreduce]
enable_local_repo: false
smoke_test_components: [hdfs, yarn, mapreduce]
```

Reply:
ok, the release version is 3.1.0 ? 3.2.0? or ?
![image](https://github.com/apache/bigtop/assets/11262916/e10c52cc-b3f2-4024-838e-5e9e0b4f715d)

Reply:
3.2.0

Reply:
add the remote repo for openeuler:
commit ffd556464a80c16bbc0b3e5eb4fa59a3f2beed42